<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSettingsPage</class>
 <widget class="QWidget" name="MainSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="exitOnLastDocClosedChk">
     <property name="text">
      <string>Exit after closing the last document</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="syncOpenDlgChk">
     <property name="text">
      <string>Sync 'Open' dialog to current doc's dir</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="singleInstanceChk">
     <property name="toolTip">
      <string>Use existing instance of application when open files</string>
     </property>
     <property name="text">
      <string>Open files in existing instance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="makeBackupChk">
     <property name="text">
      <string>Make backup copy when save file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stripSpacesChk">
     <property name="text">
      <string>Strip trailing spaces</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>188</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>singleInstanceChk</tabstop>
  <tabstop>makeBackupChk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
